src/pybind/mgr/dashboard/services/tcmu_service.py

```diff
@@ -60,7 +60,7 @@ def get_iscsi_info():
                 perf_key = "{}lock_acquired_time".format(perf_key_prefix)
                 lock_acquired_time = (mgr.get_counter(
                     'tcmu-runner', service_id, perf_key)[perf_key]
-                                      or [[0, 0]])[-1][1] / 1000000000
```

Comment:
This `autopep8` fix will raise an `pylint` error.
```
************* Module dashboard.services.tcmu_service
services/tcmu_service.py:63:0: C0330: Wrong continued indentation (add 18 spaces).
                    or [[0, 0]])[-1][1] / 1000000000
                    ^                 | (bad-continuation)
```
I didn't have find a solution to get it formatted correctly, so i added a `# noqa` to do not format it by `autopep8`.

Reply:
😟  ... I don't think we should have to manually add annotation for excluding pure indentation issues. Perhaps it's best to either try with another auto-fixed (pep8ify?) or simply tune pylint (which will probably be more tunable)... but I personally don't understand what autopep8 is doing this time (and doesn't look great either BTW).

What do you think?

Reply:
BTW [a comparison of Python autofomatters](https://www.kevinpeters.net/auto-formatters-for-python).

Reply:
And [more comparisons](https://blog.frank-mich.com/python-code-formatters-comparison-black-autopep8-and-yapf/): Black and YAPF seem the winners (in consistency at least).